include/fmi4c_private.h

```diff
@@ -27,7 +26,7 @@
 #define LOADFUNCTION2(FUNCNAME) (FUNCNAME ## _t)GetProcAddress(dll, getFunctionName(fmu->fmi1.modelName, #FUNCNAME));
 #define LOADFUNCTION(FUNCNAME) (FUNCNAME ## _t)GetProcAddress(dll, #FUNCNAME);
 #else
-#define LOADFUNCTION2(FUNCNAME) (FUNCNAME ## _t)dlsym(dll, getFunctionName(fmu->modelName, #FUNCNAME));
+#define LOADFUNCTION2(FUNCNAME) (FUNCNAME ## _t)dlsym(dll, getFunctionName(fmu->instanceName, #FUNCNAME));
```

Comment:
I believe this should be `fmu->fmi1.modelname`. But I removed the load function macros and replaced them with a function anyway.